src/Workspaces/Core/Portable/NamingStyles/EditorConfig/EditorConfigNamingStyleParser_SymbolSpec.cs

```diff
@@ -281,5 +282,206 @@ private static ImmutableArray<ModifierKind> ParseModifiers(string symbolSpecRequ
+                return "*";
+            }
+
+            return string.Join(", ", symbols.Select(symbol => symbol.ToEditorConfigString()));
+        }
+
+        private static string ToEditorConfigString(this SymbolKindOrTypeKind symbol)
+        {
+            switch (symbol.MethodKind)
+            {
+                case MethodKind.Ordinary:
+                    return "method";
+
+                case MethodKind.LocalFunction:
+                    return "local_function";
+
+                case null:
+                    break;
+
+                default:
+                    throw ExceptionUtilities.UnexpectedValue(symbol);
+            }
+
+            switch (symbol.TypeKind)
+            {
+                case TypeKind.Class:
+                    return "class";
+
+                case TypeKind.Struct:
+                    return "struct";
+
+                case TypeKind.Interface:
+                    return "interface";
+
+                case TypeKind.Enum:
+                    return "enum";
+
+                case TypeKind.Delegate:
+                    return "delegate";
+
+                case null:
+                    break;
+
+                default:
+                    throw ExceptionUtilities.UnexpectedValue(symbol);
+            }
+
+            switch (symbol.SymbolKind)
+            {
+                case SymbolKind.Namespace:
+                    return "namespace";
+
+                case SymbolKind.Property:
+                    return "property";
+
+                case SymbolKind.Field:
+                    return "field";
+
+                case SymbolKind.Event:
+                    return "event";
+
+                case SymbolKind.Parameter:
+                    return "parameter";
+
+                case SymbolKind.TypeParameter:
+                    return "type_parameter";
+
+                case SymbolKind.Local:
+                    return "local";
+
+                case null:
+                    break;
+
+                default:
+                    throw ExceptionUtilities.UnexpectedValue(symbol);
+            }
+
+            throw ExceptionUtilities.UnexpectedValue(symbol);
+        }
+
+        public static string ToEditorConfigString(this ImmutableArray<Accessibility> accessibilities, string languageName)
+        {
+            if (accessibilities.IsDefaultOrEmpty)
+            {
+                return "";
+            }
+
+            if (_allAccessibility.All(accessibilities.Contains) && accessibilities.All(_allAccessibility.Contains))
+            {
+                return "*";
+            }
+
+            return string.Join(", ", accessibilities.Select(accessibility => accessibility.ToEditorConfigString(languageName)));
+        }
+
+        private static string ToEditorConfigString(this Accessibility accessibility, string languageName)
+        {
+            switch (accessibility)
+            {
+                case Accessibility.NotApplicable:
+                    return "local";
```

Comment:
this could use a comment :)

Reply:
➡️ This is just a mirror of existing code a few lines up:

https://github.com/dotnet/roslyn/blob/7f809efa06ad3367dbac5971c9026433be6369b6/src/Workspaces/Core/Portable/NamingStyles/EditorConfig/EditorConfigNamingStyleParser_SymbolSpec.cs#L213-L215